### PR TITLE
Fixed issue with seconds per month

### PR DESCRIPTION
**Pull request by SirBriguy**
Added helper function to return number of seconds per month (kerbal or
earth as appropriate) and updated the 2 locations where it was hard
coded.

### CODE REVIEW

File: Source/USILifeSupport/LifeSupportUtilities.cs [hunk 1]
```diff
@@ -4,6 +4,14 @@ namespace LifeSupport
 {
     public static class LifeSupportUtilities
     {
+        public static double SecondsPerMonth()
+        {
+            const double secsPerMinute = 60d;
+            const double secsPerHour = secsPerMinute * 60d;
+            double secsPerDay = GameSettings.KERBIN_TIME ? secsPerHour * 6d : secsPerHour * 24d;
+            return GameSettings.KERBIN_TIME ? secsPerHour * 38.6d : secsPerDay * 30.4d;
```
Comment (Kerbas-ad-astra) [inline Source/USILifeSupport/LifeSupportUtilities.cs]: Are Kerbin months 38.6 hours long?  I think the return value should be secsPerDay*30d in both cases (i.e. no "?" necessary).

**Comment by SirBriguy** [comment]
It is according to the wiki, but I am not sure if that is out of date.

http://wiki.kerbalspaceprogram.com/wiki/Time

I will do a test tomorrow morning in game and let you know.

Perhaps just rate everything in days to eliminate confusion?

**Comment by SirBriguy** [comment]
I just tested in sandbox. It took ~6.5-7 days for the Mun to go completely
around Kerbin.

Alternatively, you could leave it at 30 day 'months', but it probably
should be stated as such someplace. Only reason I noticed was I worked on a
spreadsheet to help me plan my trips, and my numbers were not matching
because I used the shorter 'month'.

Brian

**Comment by Kerbas-ad-astra** [comment]
I guess the Munar month may be 38.6 hours long, but as a unit of time for measuring long-term space missions and space habitat lifetimes, it feels kind of small, and since colonies can go anywhere, it's not necessary to adhere strictly to Kerbin's orbital period or lunar cycle.  (I'm with you that the month should be stated somewhere.)

**Pull request merged**